packages/neuron-ui/src/utils/scriptAndAddress.ts

```diff
@@ -0,0 +1,23 @@
+import { predefined } from '@ckb-lumos/config-manager'
+import { encodeToAddress, parseAddress, generateAddress } from '@ckb-lumos/helpers'
+
+const { LINA: MAINNET, AGGRON4: TESTNET } = predefined
+
+const CONFIGS = {
+  [MAINNET.PREFIX]: MAINNET,
+  [TESTNET.PREFIX]: TESTNET,
+}
+
+export const scriptToAddress = (
+  script: CKBComponents.Script,
+  { isMainnet = true, deprecated = false }: { isMainnet?: boolean; deprecated?: boolean }
```

Comment:
```suggestion
  { isMainnet = true, version = 'CKB2021' }: { isMainnet?: boolean; version?: 'CKB2019' | 'CKB2021' }
```

Reply:
`version` makes more sense when we have multiple versions of addresses. 
e.g. `'CKB2019 | 'CKB2021' | 'CKB2025'`

Right now we just have the full new address format or the deprecated address format.
So `deprecated` is enough and it's convenient and easy to understand and passed in as param.

`true` | `false`  vs `CKB2019` | `CKB2021`

We can change this to version when CKB does have another new version of adress format.